test/sales_engine_test.rb

```diff
@@ -37,7 +37,7 @@ def test_can_override_default_data_location
   end
 
   def test_it_passes_parsed_merchant_data_to_merchant_repository
-    data_location = {merchant: "./data/fixtures/merchants.csv", item: "./data/fixtures/items.csv"}
+    data_location = {merchant: "./data/fixtures/merchants.csv", item: "./data/fixtures/items.csv", customer: "./data/fixtures/customers.csv"}
```

Comment:
Line is too long. [141/80]<br>Space inside { missing.<br>Space inside } missing.